.buildkite/commands/release-upload.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -eu
+
+"$(dirname "${BASH_SOURCE[0]}")/shared_setup.sh"
+
+echo "--- :arrow_down: Downloading Artifacts"
+ARTIFACTS_DIR='artifacts' # Defined in Fastlane, see ARTIFACTS_FOLDER
+STEP=testflight_build
+buildkite-agent artifact download "$ARTIFACTS_DIR/*.ipa" . --step $STEP
+buildkite-agent artifact download "$ARTIFACTS_DIR/*.zip" . --step $STEP
+
+echo "--- :closed_lock_with_key: Installing Secrets"
+bundle exec fastlane run configure_apply
+
+echo "--- :testflight: Uploading to TestFlight"
+bundle exec fastlane upload_app_store_connect_build_to_testflight
+
+echo "--- :arrow_up: Uploading dSYM to Sentry"
+set +e
+bundle exec fastlane symbols_upload
+SENTRY_UPLOAD_STATUS=$?
+set -e
+
+if [[ $SENTRY_UPLOAD_STATUS -ne 0 ]]; then
+  echo "^^^ +++ Failed to upload dSYM to Sentry! Make sure to download dSYM from the build step artifacts and upload manually."
+fi
```

Comment:
Given the Sentry upload has been a bit flaky recently, I thought we could trap errors and continue the build, reminding the sure to upload manually.

And, it just occurred to me that this could also be done with a build annotation.

Reply:
The "flakiness" of the dSYM upload was just due to a ["no space left on device" error](https://buildkite.com/automattic/pocket-casts-ios/builds/8869#0193447a-9f98-47c0-8418-a0624a9e024b/725-5522), which is a known issue in the `xcode-16.0-v7` VM, so it's not so much the Sentry command that has been flaky but the VM itself; updating to `xcode-16.0-v10` (which has already been done in `trunk`, just not in `release/*` yet) should fix the issue in the short term already.

That being said, yeah it wouldn't hurt to make it a `buildkite-agent annotate`. I'll take note of it for future iterations 👍